apis_bibsonomy/models.py

```diff
@@ -29,3 +32,25 @@ def bibtexjson(self):
 
     def get_absolute_url(self):
         return reverse("apis_bibsonomy:referencedetail", kwargs={"pk": self.pk})
+
+    def save(self):
+        bib_ref = self.bibs_url
```

Comment:
bib_ref is defined here but not used (you reference self.bibs_url beneath, where you could use bib_ref.